R/pkg/R/DataFrame.R

```diff
@@ -932,7 +932,7 @@ setMethod("sample_frac",
 #' @param x a SparkDataFrame.
 #' @family SparkDataFrame functions
 #' @rdname nrow
-#' @name count
+#' @name nrow
```

Comment:
hmm, I don't think we should change @name - this should match the function name, "count"